repo2docker/buildpacks/r.py

```diff
@@ -226,6 +235,22 @@ def get_build_scripts(self):
             ),
         ]
 
+        if self.stencila_manifest_contexts:
```

Comment:
If we make the default value an empty set we can skip this level of indention and directly test if R is a detected language.